ui-tests/tests/jupytereverywhere.spec.ts

```diff
@@ -125,13 +153,15 @@ test.describe('General', () => {
         id: notebookId,
         domain_id: 'domain',
         readable_id: null,
-        content: TEST_NOTEBOOK
+        content: PYTHON_TEST_NOTEBOOK
       };
       await route.fulfill({ json });
     });
 
     await page.goto(`lab/index.html?notebook=${notebookId}`);
 
+    await page.waitForTimeout(5000); // JupyterLab rotating icon is taking a while to disappear, so just to be safe
```

Comment:
This is why we elsewhere we use:

```suggestion
    await page.waitForSelector('.jp-LabShell');
```

Using explicit timeout especially this large is discouraged as it is easy to make tests pass with them but then this quickly compounds and suddenly they take half an hour to run ;)